### DISCUSSION TITLE
Flow Rate

### DISCUSSION THREAD

**Discussion by jisr007** [Jyers/Marlin#43] [Q&A] [answered]
The flow rate option respects the flow imposed by Cura or changes it back to 100% automatically and you have to change it back to the flow rate with which my ender 3 v2 works?

**Comment by Schnabulation** [comment]
...I think it is just a multiplier of the flow you set in your slicer. So it will stay on 100% - even if you change it up in Cura. But if you change it on the display it will multiply with the flow you set in Cura.

**Reply by Jyers** [reply]
The overall flow set in Cura is a strange thing as it doesn't seem to modify the flowrate in marlin. However if any commands are executed that change the flow rate, either in the gcode or from a host, the flow on the screen will react to that change.

**Reply by jisr007** [reply]
Muchas gracias por la respuesta.

**Comment by Jyers** [comment, marked as answer]
The overall flow set in Cura is a strange thing as it doesn't seem to modify the flowrate in marlin. However if any commands are executed that change the flow rate, either in the gcode or from a host, the flow on the screen will react to that change.
